### ISSUE TITLE
Link to WCAG 2.1 instead of 2.0?

### ISSUE THREAD

**Issue by ericwbailey** [a11yproject/a11yproject.com#1460]
### Discussed in https://github.com/a11yproject/a11yproject.com/discussions/1459

<div type='discussions-op-text'>

<sup>Originally posted by **robinmetral** August 10, 2022</sup>
Can we link to the WCAG 2.1 website instead of WCAG 2.0? I'm not sure if this is consistent throughout the site, but I'm seeing it on e.g. the checklist links to success criteria. For instance, users are sent to [3.1.5 Reading Level (WCAG 2.0)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-supplements.html) instead of [3.1.5 Reading Level (WCAG 2.1)](https://www.w3.org/WAI/WCAG21/Understanding/reading-level.html).

Am I missing a reason why these should continue to link to 2.0?</div>

**Comment by SaptakS** [comment]
@ericwbailey I can take over this issue.

**Comment by SaptakS** [comment]
Seems like WCAG 2.2 is [releasing on 5th October](https://www.w3.org/2023/10/03-ag-minutes.html)! I will update the links to WCAG2.2
